### PR TITLE
Add SameSite=none to cookie

### PR DESCRIPTION
**Pull request by DGarbar**
https://github.com/prebid/prebid-server/issues/900
- Added `SameSite=none` to `Set-Cookie` header.
- Implemented through Headers without vertx.Cookie.addCookie(), bc Vertx don`t have special method for that case.

### CODE REVIEW

File: src/main/java/org/prebid/server/util/HttpUtil.java [hunk 1]
```diff
@@ -150,4 +150,8 @@ public static String getDomainFromUrl(String url) {
         return context.cookies().stream()
                 .collect(Collectors.toMap(Cookie::getName, Cookie::getValue));
     }
+
+    public static String cookieToSetCookie(Cookie cookie) {
```
Comment (RossGoncharuk) [inline src/main/java/org/prebid/server/util/HttpUtil.java]: Please rename to `cookieToHeaderValue`, this name is confusing

File: src/test/java/org/prebid/server/util/HttpUtilTest.java [hunk 2]
```diff
@@ -133,4 +133,18 @@ public void cookiesAsMapShouldReturnExpectedResult() {
         assertThat(cookies).hasSize(1)
                 .containsOnly(entry("name", "value"));
     }
+
+    @Test
+    public void cookieToSetCookieShouldReturnCookieWithAdditionalParameter() {
```
Comment (RossGoncharuk) [inline src/test/java/org/prebid/server/util/HttpUtilTest.java]: Please rename test to `cookieToHeaderValueShouldReturnExpectedString` and `setCookie` variable to `setCookieHeader`. Thanks!

File: src/main/java/org/prebid/server/handler/SetuidHandler.java [hunk 3]
```diff
@@ -166,6 +166,12 @@ private void respondWithCookie(RoutingContext context, String bidder, UidsCookie
                 .build());
     }
 
+    private void addCookie(RoutingContext context, Cookie cookie) {
+        if (cookie != null) {
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/handler/SetuidHandler.java]: Is it possible null here?

File: src/main/java/org/prebid/server/util/HttpUtil.java (hunk 1 above)
Comment (rpanchyk) [reply src/main/java/org/prebid/server/util/HttpUtil.java]: or m.b. something like `toSetCookieHeaderValue`. What do you think guys?

Comment (RossGoncharuk) [reply src/main/java/org/prebid/server/util/HttpUtil.java]: `resolveSetCookieHeaderValue`?

**Pull request merged**